examples/subplots_axes_and_figures/multiple_figs_demo.py

```diff
@@ -1,9 +1,20 @@
 """
-==================
-Multiple Figs Demo
-==================
+===================================
+Managing multiple figures in pyplot
+===================================
+
+`matplotlib.pyplot` uses the concept of a *current figure* and *current axes*.
+Figures are identified via a figure number that is passed to `~.pyplot.figure`.
+The figure with the given number is set as *current figure*. Additionally, if
+no figure with the number exists, a new one is created.
+
+.. note::
+
+    We discourage working with multiple figures in pyplot because managing
+    the *current figure* is cumbersome and error-prone. Instead, we recommend
+    to use the object-oriented approach and call methods on figure and Axes
```

Comment:
This is definitely talking about the class.
```suggestion
    to use the object-oriented approach and call methods on Figure and Axes
```